assets/javascripts/kitten/components/cards/reward-card.js

```diff
@@ -0,0 +1,400 @@
+import { Marger } from 'kitten/components/layout/marger'
+import {
+  Grid as GridBase,
+  GridCol as GridColBase,
+} from 'kitten/components/grid/grid'
+import { Button as ButtonBase } from 'kitten/components/buttons/button'
+import { Title as TitleBase } from 'kitten/components/typography/title'
+import { Text as TextBase } from 'kitten/components/typography/text'
+import { Paragraph as ParagraphBase } from 'kitten/components/typography/paragraph'
+import { IconBadge as IconBadgeBase } from 'kitten/components/notifications/icon-badge'
+import { CheckedIcon } from 'kitten/components/icons/checked-icon'
+import { HorizontalStroke as HorizontalStrokeBase } from 'kitten/components/layout/horizontal-stroke'
+import COLORS from 'kitten/constants/colors-config'
+import { parseHtml } from 'kitten/helpers/utils/parser'
+import { ScreenConfig } from 'kitten/constants/screen-config'
+import { mediaQueries } from 'kitten/hoc/media-queries'
+
+const Grid = Radium(GridBase)
+const GridCol = Radium(GridColBase)
+const Button = Radium(ButtonBase)
+const Title = Radium(TitleBase)
+const Text = Radium(TextBase)
+const Paragraph = Radium(ParagraphBase)
+const IconBadge = Radium(IconBadgeBase)
+const HorizontalStroke = Radium(HorizontalStrokeBase)
+
+class RewardCardBase extends Component {
+  static propTypes = {
+    titleMount: PropTypes.string.isRequired,
+    titleDescription: PropTypes.string.isRequired,
+    textDescription: PropTypes.string.isRequired,
+
+    titleContributors: PropTypes.string,
+    titleSmallContributors: PropTypes.string,
+    titleDelivery: PropTypes.string,
+    titleSmallDelivery: PropTypes.string,
+    titleAvailability: PropTypes.string,
+    titleSmallAvailability: PropTypes.string,
+    valueContributors: PropTypes.string,
+    valueDelivery: PropTypes.string,
+    valueAvailability: PropTypes.string,
+
+    button: PropTypes.string,
+
+    myContribution: PropTypes.string,
+    manageContribution: PropTypes.string,
+
+    imageSrc: PropTypes.string.isRequired,
+    imageSrcSmall: PropTypes.string,
+  }
+
+  static defaultProps = {
+    imageProps: {
+      src: 'https://placehold.it/350x200/caf4fe/caf4fe',
+      alt: '',
+    },
+  }
+
+  removeCurrentFocus = () => {
+    document.activeElement.blur()
+  }
+
+  render() {
+    const styleCard = [
+      styles.card,
+      this.props.isDisabled && styles.card.isDisabled,
+    ]
+
+    return (
+      <StyleRoot>
+        <Grid>
+          <GridCol>
+            <div style={styleCard} disabled={this.props.isDisabled}>
+              <Marger bottom={this.props.viewportIsSOrLess ? 0 : 5}>
+                <Grid style={styles.card.addPadding}>
+                  <GridCol col-m="7">{this.renderDescription()}</GridCol>
+                  <GridCol col-m="4" offset-m="1">
+                    {this.renderImage()}
+                  </GridCol>
+                </Grid>
+                {this.props.viewportIsSOrLess && (
+                  <Fragment>{this.renderChoiceButton()}</Fragment>
```

Comment:
Pourquoi est-ce qu'on a besoin d'un `<Fragment />` ici ?
J'ai l'impression qu'on pourrait s'en passer.

Reply:
Parce que sinon j'ai une erreur. Tu peux pas mettre un `render` sans l'entourer d'une `<div>`  dans une condition.

Reply:
Ok ça me va 👍

Reply:
Hum c'est bizarre en effet ce `<Fragment>`.

T'as essayé un truc comme ca : 
```jsx
{this.props.viewportIsSOrLess && this.renderChoiceButton()}
```

Reply:
Sinon l'autre solution est un early return du côté de `renderChoiceButton` qui embarque la condition sur `this.props.viewportIsSOrLess`.